csrc/kernel_cache.cpp

```diff
@@ -1200,17 +1200,7 @@ std::vector<at::Tensor> FusionKernelRuntime::runKernelWithInput(
     executor.setMeasureKernelTimeFlag(true);
   }
 
-  if (isProfilerEnabled()) {
-    auto& sprof = FusionProfiler::segment(group_id);
-    sprof.inputBytesAccessed(executor.inputBytesProcessed(args));
-    sprof.startKernel(args.getDeviceIndex());
-  }
   auto outputs = executor.runFusion(args, launch_params, compile_params);
-  if (isProfilerEnabled()) {
-    auto& sprof = FusionProfiler::segment(group_id);
-    sprof.stopKernel();
-    sprof.outputBytesAccessed(executor.outputBytesProcessed(outputs));
-  }
```

Comment:
This would capture times for executors which do not have a compiled kernel as well, but it seems we lose this now?